specificatie/openapi.yaml

```diff
@@ -499,11 +501,12 @@ components:
       description: "Zoekterm op postcode, woonplaats, straatnaam, huisnummer, huisletter, huisnummertoevoeging."
       name: zoek
       in: query
-      required: true
+      required: false
```

Comment:
zie mijn opmerking in het issue https://github.com/VNG-Realisatie/Haal-Centraal/issues/74#issuecomment-811805417

1. op deze plek wil je het wel required houden
2. de nieuwe zoek-parameter moet ook minLength: 0 en maxLength: 255 krijgen

Reply:
De zoek parameter kan hier niet verplicht zijn, omdat er bij dit /adressen endpoint meerdere mogelijkheden zijn om te zoeken:
1.  met een query parameter pand identificatie OF
2. met een query parameter adresseerbaar object identificatie OF
3. met 'fuzzy search' / zoek query parameter
Als zoek verplicht zou worden dan zou bij het zoeken met pand identificatie of adresseerbaar object identificatie ook altijd een zoek parameter opgegeven moeten worden en dat lijkt me niet de bedoeling. 
Als zoek een verplichte query parameter moet worden, dan is er maar één mogelijkheid: een apart (nieuwe) endpoint specificeren.